app/src/main/java/com/pvp/app/ui/screen/settings/SettingsScreen.kt

```diff
@@ -64,7 +65,40 @@ private fun SettingNotificationReminderMinutes(
             }
         },
         title = "Set Reminder Time",
-        value = "$minutes minute(s)"
+        value = "$minutes minute${if (minutes != 1) "s" else ""}"
+    )
+}
+
+@Composable
+private fun SettingCupVolumeMl(
+    model: SettingsViewModel = hiltViewModel()
+) {
+    val rangeCupVolume = remember { model.fromConfiguration { it.rangeCupVolume } }
+    val volume by model
```

Comment:
Empty line required above volume, because of multi-line code block used for initialization.